### PR TITLE
ddl: refine syncer log and add some comments

### PR DESCRIPTION
**Pull request by winkyao**
@zimulala PTAL

### CODE REVIEW

**Comment by shenli** [comment]
LGTM

File: ddl/ddl_worker.go [hunk 1]
```diff
@@ -373,10 +373,13 @@ func (d *ddl) waitSchemaChanged(ctx goctx.Context, waitTime time.Duration, lates
 	if err != nil {
 		log.Infof("[ddl] update latest schema version %d failed %v", latestSchemaVersion, err)
 		if terror.ErrorEqual(err, goctx.DeadlineExceeded) {
+			// If err is goctx.DeadlineExceeded, means waitTime(2 * lease) is elapsed. so all the schema is synced by ticker,
+			// no need to use etcd to sync, we return directly.
 			return
 		}
 	}
 
+	// OwnerCheckAllVersions return only when goctx is time out(2 * lease) or all TiDB schema is synced.
```
Comment (zimulala) [inline ddl/ddl_worker.go]: s/return/returns
s/all TiDB schema is/all TiDB schemas are

File: ddl/ddl_worker.go [hunk 2]
```diff
@@ -373,10 +373,13 @@ func (d *ddl) waitSchemaChanged(ctx goctx.Context, waitTime time.Duration, lates
 	if err != nil {
 		log.Infof("[ddl] update latest schema version %d failed %v", latestSchemaVersion, err)
 		if terror.ErrorEqual(err, goctx.DeadlineExceeded) {
+			// If err is goctx.DeadlineExceeded, means waitTime(2 * lease) is elapsed. so all the schema is synced by ticker,
```
Comment (zimulala) [inline ddl/ddl_worker.go]: Ditto.

**Comment by zimulala** [comment]
PTAL @queenypingcap

File: ddl/syncer.go [hunk 3]
```diff
@@ -279,13 +279,14 @@ func (s *schemaVersionSyncer) OwnerCheckAllVersions(ctx goctx.Context, latestVer
 	}()
 	for {
 		if isContextDone(ctx) {
+			// ctx is canceld or timeout.
```
Comment (zimulala) [inline ddl/syncer.go]: s/canceld/canceled

File: ddl/ddl_worker.go [hunk 4]
```diff
@@ -373,10 +373,13 @@ func (d *ddl) waitSchemaChanged(ctx goctx.Context, waitTime time.Duration, lates
...
+			// no need to use etcd to sync, we return directly.
```
Comment (queenypingcap) [inline ddl/ddl_worker.go]: -> There is no need to use etcd to sync. The function returns the error directly.

File: ddl/ddl_worker.go (hunk 1 above)
Comment (queenypingcap) [reply ddl/ddl_worker.go]: I agree with Lynn.

**Comment by winkyao** [comment]
PTAL @queenypingcap @zimulala

**Pull request merged**